inclusive-design/wecount.inclusivedesign.ca
# feat: add service worker for caching and PWA support

## Description

Add service worker for caching and PWA support.

## Steps to test

1. Open the deploy preview;
2. Use dev tool to set "network" to "offline" so that the browser works in the offline mode;
3. Click around; webpages are still accessible;
4. Run lighthouse report;
5. PWA item should have been enabled.
